src/App.vue

```diff
@@ -1,9 +1,9 @@
 <template>
   <div id="app">
     <Header/>
-    <SubHeading/>
+    <SubHeading v-if="!isNominate"/>
```

Comment:
Just move this component to the developer list
```suggestion
    <SubHeading/>
```

Reply:
Done  .. .